public/js/Player.js

```diff
@@ -13,12 +13,10 @@ let rightKey;
 let active = true;
```

Comment:
moved all the exports to a object at the bottom of this file and added the variables that are needed between files too